src/v2/Components/__tests__/LotTimer.jest.tsx

```diff
@@ -98,20 +99,22 @@ describe("extendedBiddingInfoCopy", () => {
       )
     })
     describe("a bid has extended the auction", () => {
+      beforeEach(() => {
+        Settings.now = jest.fn(() =>
+          new Date("2022-03-08T12:33:37.000Z").getTime()
+        )
+      })
```

Comment:
cc @mzikherman @damassi 

@nspinazz89 and I paired on this a bit and came up with this approach. We are manipulating [Settings](https://moment.github.io/luxon/api-docs/index.html#settings); that's suggested by the luxon docs.

Reply:
I wonder if y'all might need `jest.resetAllMocks()` in an afterEach in the future

Reply:
I wondered about that, but I'm hoping `jest.fn` would handle cleanup? Presumably, we'll notice issues in other tests if this breaks things.

Reply:
yeah i do think you're right